src/debug/ee/amd64/doc/README.md

```diff
@@ -0,0 +1,11 @@
+The files in this folders were used to create `amd64InstrDecode.h`
```

Comment:
I think we should add a little more descriptive text about what we are doing here and why we chose this approach. I don't think it will be obvious to everyone what the workflow is doing even if following the steps below does mechanically carry it out.

If there is a high level description of AMD64 assembly that would let someone unfamiliar with its encoding (such as me) understand a little better that would be great, or if the best available explanation is "go read the AMD instruction manual" then a link to the manual and references to the relevant sections would be handy.

Reply:
The encoding is kind of complicated.  Citing the manual seems best.  I added a reference to the encoding flow chart and the manual section to the readme.  I think if I try to summarize it I will just create confusion.